src/fontique/backend/android.rs

```diff
@@ -109,15 +164,37 @@ impl SystemFonts {
             name_map: Arc::new(name_map),
             generic_families: Arc::new(generic_families),
             family_map,
+            locale_fallback: locale_fallback.into(),
+            script_fallback: script_fallback.into(),
         }
     }
 
-    pub fn family(&mut self, id: FamilyId) -> Option<FamilyInfo> {
+    pub fn family(&self, id: FamilyId) -> Option<FamilyInfo> {
         self.family_map.get(&id).cloned()
     }
 
-    pub fn fallback(&mut self, _key: impl Into<FallbackKey>) -> Option<FamilyId> {
-        // FIXME: This is a stub
-        Some(self.generic_families.get(GenericFamily::SansSerif)[0])
+    pub fn fallback(&self, key: impl Into<FallbackKey>) -> Option<FamilyId> {
+        let key: FallbackKey = key.into();
+        let script = key.script();
+
+        key.locale()
+            .and_then(|li| {
+                self.locale_fallback
+                    .iter()
+                    .find(|(lid, _)| li == lid.to_string())
```

Comment:
Maybe `find_map` with `some` here?

Reply:
I think in this case that doesn't have any compiler or readability benefit. It would end up being an `if` explicitly returning `Some(...)` or `None`.